bin/aws-secrets-init-resources

```diff
@@ -15,6 +15,9 @@ app=$1
 
 [[ "$app" =~ ^[a-z0-9_]+$ ]] || die "The application name should consist only of a-z, 0-9 or _ charaters."
 
+major=`aws --version 2>&1 | awk '{print $1}' | sed 's|aws-cli/1.\([0-9]*\).*$|\1|'`
+[ ! -z "$major" ] && [ $major -lt 9 ] && die "aws-cli version needs to be >= 1.9"
```

Comment:
I think this could be `>= 1.8`, unless you have some other reason that we need `>= 1.9` besides the jmespath errors.